doc/unused/specification_sheet.md

```diff
@@ -34,12 +34,12 @@ The following subitems are to be defined as functional requirements to be fulfil
 - schools
    * school id
    * name
-   * board
```

Comment:
let's remove this as this is in danger of always being out of date. I copied it to our drive folder to keep it archived